Makefile

```diff
@@ -95,6 +96,7 @@ e2e-test-docker-image: deploy/e2e-test-image/Dockerfile
 
 .PHONY: e2e-test
 e2e-test: deps e2e-test-docker-image ## Run e2e tests
+	@docker image ls
 	@go test -count=1 ./tests/e2e -v -args -dockerImage="$(E2E_TEST_DOCKER_IMAGE)"
```

Comment:
This pull requests breaks our e2e ci pipeline. Currently we are building a local image from a Dockerfile. Then we create an environment with this image:
https://github.com/openHPI/poseidon/blob/85895e17075d23f7581a2647cf1da0edae48d37f/tests/e2e/e2e_test.go#L77-L88
But we just introduced the force_pull flag there so local images are ignored and so the image can not be found.

We could solve this issue for example by using an image from a registry instead of building our own.

Reply:
Ah, that's a good idea and would also increase the e2e execution time. What if we just create a Docker image, push it to this repository's registry, and use the (static) image in our tests?

I would recommend tracking that in another PR first (decoupled from this one), merging the other, and then just rebasing here.